src/app/components/navigation/navbar/navbar.component.html

```diff
@@ -65,19 +75,13 @@
             } @else {
               @if (isAdmin$ | async) {
                 <a
```

Comment:
The admin button could also be move to the dropdown

Reply:
ok, done